### DISCUSSION TITLE
Current version (v2.7.6) not available in snap store

### DISCUSSION THREAD

**Discussion by AEndrson** [keepassxreboot/keepassxc#10013] [Q&A] [answered]
After learning version 2.7.6 is available, I checked the snap store for an updated package, and the store shows v2.7.6 in the beta channel, but after installing keepassxc from the beta channel, I checked the version, and it reports v2.8.0.

Are there plans to make v2.7.6 available in the snap store in the stable channel?

**Comment by droidmonkey** [comment, marked as answer]
Thanks for the heads up, looks like launchpad was failing to build, all fixed now.

**Reply by AEndrson** [reply]
Thanks, available and working now.  :+1:
